docker-compose.test.yml

```
@@ -58,3 +64,5 @@
         condition: service_healthy
     volumes:
       - ./src/test/load:/loadTests
+    networks:
+      - app-net
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)